.ai/auto-review/env_vars.py

```diff
@@ -30,8 +30,7 @@ def __init__(self):
             "token" : self.token,
             "base_ref" : self.base_ref,
             "pull_number" : self.pull_number,
-            "chat_gpt_token" : self.chat_gpt_token,
-            "chat_gpt_model" : self.chat_gpt_model,
+            "gemini_ai_token" : self.gemini_ai_token
         }
 
     def check_vars(self):
```

Comment:
36 : Unhandled exception if `TARGET_EXTENSIONS` is not set : Add check for `self.target_extensions` being None before splitting.  `self.target_extensions = os.getenv('TARGET_EXTENSIONS') if self.target_extensions is not None: self.target_extensions = [lang.strip() for lang in self.target_extensions.split(",")] else: raise ValueError(f"Please specify TARGET_EXTENSIONS. Coma separated, could be, like: kt,java,py,js,swift,c,h. Only these files will be reviewed")`